node/pkg/allocateip/allocateip.go

```diff
@@ -683,7 +683,7 @@ func removeHostTunnelAddr(ctx context.Context, c client.Interface, nodename stri
 		_, updateError = c.Nodes().Update(ctx, node, options.SetOptions{})
 		if _, ok := updateError.(cerrors.ErrorResourceUpdateConflict); ok {
 			// Wait for a second and try again if there was a conflict during the resource update.
-			logCtx.Infof("Error updating node %s: %s. Retrying.", node.Name, err)
+			logCtx.Infof("Error updating node %s: %v. Retrying.", node.Name, err)
```

Comment:
Could you adjust this to use `WithError`, like so:

```
			logCtx.WithError(err).Infof("Error updating node %. Retrying.", node.Name)
```

Reply:
Thanks, Done.